### DISCUSSION TITLE
Web-based editor shortcut doesn't work with French layout

### DISCUSSION THREAD

**Discussion by oncleben31** [community/community#6162] [Other Feature Feedback, Questions, & Ideas] [answered]
According to documentation the shortcut to start the web-based editor is `.`.

Today on my MacBook Pro with a French Layout. I didn't find a way to start the web-based editor in Safari of Chrome.

**Comment by bamurtaugh** [comment, marked as answer]
Hi @oncleben31, thanks for reporting! Is `.` reserved for something else in your current layout? Also, you should still be able to access github.dev by changing the github.com URL to `.dev` for any repo, branch, or PR. Please let me know if that makes sense or you think this could be made clearer in the documentation.

**Reply by oncleben31** [reply]
This is more a workaround that can unlock the situation. This is now in the documentation. I think you should at least add a warning in the documentation saying this shortcut could not work according to local keyboard layout. And perhaps propose a second shortcut that could work in other layouts.

**Reply by bamurtaugh** [reply]
Thanks for the feedback - I've opened https://github.com/github/docs/pull/13682 to add a note on the shortcut potentially not working.
